src/lib/serviceRoutines/postQueryContext.cpp

```diff
@@ -255,6 +255,8 @@ static bool queryForward
   //
   std::string     out;
   int             r;
+  std::string     url;
+  char            portV[STRING_SIZE_FOR_INT];
```

Comment:
Where this portV variable is set? Maybe you need something like:

```
snprintf(portV, sizeof(portV), "%d", port);
```

Please have a look to similar code at senderThread.cpp and check how it is done there.

Reply:
Added portV variable in line number 270 in src/lib/serviceRoutines/postQueryContext.cpp and similarly in line number 208 src/lib/serviceRoutines/postUpdateContext.cpp

Please check my latest commit https://github.com/telefonicaid/fiware-orion/pull/3749/commits/d4e75f554d1327a17ec1235529378567fd5c4454 and let me know if I am missing something.

Reply:
You are right. I missed that `sprintf` in you last commit, sorry...

NTC